kata-webhook/README.md

```diff
@@ -0,0 +1,57 @@
+# Kata Admission controller webhook
+
+Implement a simple admission controller webhook to annotate pods with the 
+kata runtime class.
+
+## How to use the admission controller
+
+```
+docker build -t mcastelino/kubewebhook-pod-annotate-example:latest
+./create_certs.sh
+```
+
+Note: The docker image location can be changed and the image needs to be
+published if the webhook needs to work.
+
+On a single machine cluster change the `imagePullPolicy` to use the locally 
+built image.
+
+
+## Making Kata the default runtime
+
+Today in `crio.conf` runc is the default runtime when a user does not specify
+`runtimeClass` in the pod spec. If you want to run a cluster where kata is used
+by default, except for workloads we know for sure will not work with kata, use
+the [admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks)
+and sample admission controller we created by running -
+
+`kubectl apply -f deploy/`
```

Comment:
You should not need a sudo if things are setup correctly

Reply:
What sort of setup is required? Adding current user to a specific group? Do we need to document that here?

Another consideration is whether we want to / can test this README doc in the CI? If that is a possibility (ideally it would be so we can guarantee the instructions are correct and continue to be so), that might require further changes such as using explicit bash code blocks: https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks

Reply:
There is no setup required to run this except to have a k8s cluster running.

Reply:
@chavafg will need your help to integrate this into the CI testing.